bids-validator/utils/files/generateMergedSidecarWithPath.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Generate Merged Sidecar Dictionary
+ *
+ * Takes an array of potential sidecards and a
+ * master object dictionary of all JSON file
+ * content and returns a merged dictionary
+ * containing all values from the potential
+ * sidecars and the sidecarName
+ */
+function generateMergedSidecarDictWithPath(potentialSidecars, jsonContents) {
```

Comment:
Just a curiosity, it is ok if the function and the name of the file are not the same?